### DISCUSSION TITLE
Not able to set it up on my local environment.

### DISCUSSION THREAD

**Discussion by Neer-7303** [provectus/kafka-ui#4086] [Q&A] [answered]
Followed the documentation but when I run kafka-ui-api I get the following error on hitting localhost:8080.

{"code":5000,"message":"class path resource [static/index.html] cannot be opened because it does not exist","timestamp":1691057745935,"requestId":"b1fc16be-1","fieldsErrors":null,"stackTrace":"java.io.FileNotFoundException: class path resource [static/index.html] cannot be opened because it does not exist\n\tat org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:211)\n\tSuppressed: The stacktrace has been enhanced by Reactor, refer to additional information below: \nError has been observed at the following site(s):\n\t*__checkpoint ⇢ com.provectus.kafka.ui.config.CorsGlobalConfiguration$$Lambda$843/0x00000008011bb180 [DefaultWebFilterChain]\n\t*__checkpoint ⇢ com.provectus.kafka.ui.config.CustomWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ com.provectus.kafka.ui.config.ReadOnlyModeFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ AuthorizationWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ ExceptionTranslationWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ LogoutWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ ServerRequestCacheWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ SecurityContextServerWebExchangeWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ ReactorContextWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ HttpHeaderWriterWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ ServerWebExchangeReactorContextWebFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ org.springframework.security.web.server.WebFilterChainProxy [DefaultWebFilterChain]\n\t*__checkpoint ⇢ org.springframework.web.filter.reactive.ServerHttpObservationFilter [DefaultWebFilterChain]\n\t*__checkpoint ⇢ HTTP GET \"/\" [ExceptionHandlingWebHandler]\nOriginal Stack Trace:\n\t\tat org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:211)\n\t\tat com.provectus.kafka.ui.util.ResourceUtil.readAsString(ResourceUtil.java:16)\n\t\tat

**Comment by Haarolean** [comment, marked as answer]
Are you running a self-built jar? It seems like you've built one with 'local' profile rather than 'prod'.
